backendRoot/src/test/java/com/virtuslab/gitmachete/backend/integration/GitMacheteStatusTest.java

```diff
@@ -151,16 +151,16 @@ private void printCommonParts(BaseGitMacheteBranch branch, int level, StringBuil
       sb.append("  ");
       sb.append(customAnnotation.get());
     }
-    var originSync = branch.getSyncToOriginStatus();
-    if (originSync != SyncToOriginStatus.InSync) {
+    var originSync = branch.getSyncToRemoteStatus();
+    if (originSync.getStatus() != SyncToRemoteStatus.Status.InSync) {
       sb.append(" (");
-      if (originSync == SyncToOriginStatus.Ahead)
+      if (originSync.getStatus() == SyncToRemoteStatus.Status.Ahead)
```

Comment:
`Match.Case` maybe (Java 12 has switch expressions... but as for now we're stuck at Java 11 anyway coz there isn't any JBR for Java >11)